cli/reason_classifier_cli.py

```diff
@@ -0,0 +1,202 @@
+"""
+CLI Interface for Healthcare Reason Classification
+
+This provides a command-line interface for testing and using the 
+healthcare reason classifier system with real healthcare data.
+"""
+
+import argparse
+import json
+import sys
+from pathlib import Path
+
+# Add project root to path
+REPO_ROOT = Path(__file__).resolve().parents[1]
+if str(REPO_ROOT) not in sys.path:
+    sys.path.insert(0, str(REPO_ROOT))
+
+def classify_single_query(query: str):
```

Comment:
I think we should probably move these to `classifier/utils.py` as well, and possibly even merge directly with the ones from `healthcare_classifier_cli.py` so we have just a single entry point to the entire classifier pipeline.  I suppose splitting them like this makes it easier to test each in isolation, but for the final project I think it would be desirable if we had a single entrypoint to the entire classification pipeline.